packages/ui/public/locales/en/translation.json

```
@@ -3,6 +3,10 @@
   "30 minutes": "",
   "5 minutes": "",
   "A multichain crypto wallet for Polkadot & Kusama ecosystem": "A multichain crypto wallet <br /> for <strong>Polkadot & Kusama</strong> ecosystem",
+  "Account name": "",
+  "Account name should not exceed 16 characters": "",
+  "Account removed": "{{account.name}} removed",
```

I think we should use this instead of above as when doing `t<string>('{{account.name}} removed')` we could see exactly what param we should pass without having to check the translation file.
```suggestion
  "{{account.name}} removed"": "",
```